core/src/main/java/org/mapfish/print/map/style/json/JsonStyleParserHelper.java

```diff
@@ -766,16 +768,27 @@ String getGraphicFormat(final String externalGraphicFile, final PJsonObject styl
         String mimeType = null;
         if (!Strings.isNullOrEmpty(styleJson.optString(JSON_GRAPHIC_FORMAT))) {
             mimeType = styleJson.getString(JSON_GRAPHIC_FORMAT);
+            mimeType = toSupportedMimeType(mimeType);
         } else {
             int separatorPos = externalGraphicFile.lastIndexOf(".");
 
             if (separatorPos >= 0) {
                 mimeType = "image/" + externalGraphicFile.substring(separatorPos + 1).toLowerCase();
-            } else {
-                mimeType = "";
+                mimeType = toSupportedMimeType(mimeType);
+            }
+            if (mimeType.equals("")) {
+                try {
+                    URLConnection connexion = new URL(externalGraphicFile).openConnection();
```

Comment:
Should use the same stuff we use for getting tiles. That way you go through all the stuff specified in the YAML for the URL rewrites, headers, proxies, ...